tests/unit/cypher_main_visitor.cpp

```diff
@@ -4124,6 +4124,19 @@ TEST_P(CypherMainVisitorTest, VersionQuery) {
   ASSERT_NO_THROW(ast_generator.ParseQuery("SHOW VERSION"));
 }
 
+TEST_P(CypherMainVisitorTest, ConfigQuery) {
```

Comment:
I think this is nice but plz add an `e2e` test to confirm the expected behavior, that is, that the values returned are the config values used when memgraph was launched,